### ISSUE TITLE
HL-L2460DW working with HL-L2370DN driver

### ISSUE THREAD

**Issue by russsharek** [pdewacht/brlaser#216]
Despite being a networked printer, the HL-L2660DW required a cups+brlaser installation to get functional. :disappointed: 

Using the HL-L2370DN driver, I was able to print various documents as expected.

Duplex printing is still glitchy.

**Comment by QORTEC** [comment]
Could you please connect your printer to your computer via USB and run the following command:
`sudo lpinfo --include-schemes usb -l -v`

By providing the output of the above command we we will be able to create a entry for your specific printer model in the driver.

**Comment by russsharek** [comment]
I get no output from the command.

**Comment by QORTEC** [comment]
Was your printer on and connect using USB, when you ran the command?

**Comment by russsharek** [comment]
Yes it was, though nothing I could do would make it show up on both Debian and OpenBSD.

Happy to debug further, though I admit I'm out of my depth.

**Comment by QORTEC** [comment]
I have a pending update to [README.md](https://github.com/Owl-Maintain/brlaser/blob/README.net/README.md#testing-other-printers) on `Owl-Maintain/brlaser`; 
I've added alternate (network) command that you can run.

The relevant section for you would be:

> With your printer powered on, connect it to your computer via USB or obtain your printer's IP address.
> 
> **For network printers:**
> - Run the following command:
>   
>    ``/usr/lib/cups/backend/snmp IP_ADDRESS_HERE``
> 
> - Example of output:
>    ```
>    INFO: Using default SNMP Community public
>    network lpd://HL-2270DW-LAN/BINARY_P1 "Brother HL-2270DW series" "Brother HL-2270DW series" "MFG:Brother;CMD:PJL,PCL,PCLXL;MDL:HL-2270DW series;CLS:PRINTER;CID:Brother Laser Type1;" ""
>    ```
> 
> Please provide the output of the command so that we can add the proper entry for your specific printer model in the driver.

Does the section still need to be improved, or are the instructions clear as they currently are?

**Comment by russsharek** [comment]
The updated documentation was very clear. Thank you!

Here's the output of the snmp command, run from an OpenBSD client on the network:

>  INFO: Using default SNMP Community public
network lpd://BRN94DDF8078B73/BINARY_P1 "Brother HL-L2460DW" "Brother HL-L2460DW" "MFG:Brother;CMD:PJL,PCL,PCLXL,URF;MDL:HL-L2460DW;CLS:PRINTER;CID:Brother Laser Type1;URF:W8,CP1,IS4-1,MT1-3-4-5-8,OB10,PQ3-4-5,RS300-600-1200,V1.5,DM1;" ""

**Comment by QORTEC** [comment]
Thanks!

It looks like OpenBSD now uses `Owl-Maintain/brlaser`. 

I've made a note to add your printer there, you should see your printer in the next update (`v6.2.8`).

**Comment by russsharek** [comment]
Thank you!
